test/manipulation.test.js

```diff
@@ -1552,6 +1567,41 @@ describe('manipulation', function() {
         done();
       });
     });
+
+    it('correctly coerces the PK value', async () => {
```

Comment:
AFAICT, this test was not present in https://github.com/strongloop/loopback-datasource-juggler/pull/1790. I am not entirely opposed to add it, but would like to understand your motivation & reasoning first.

Reply:
@bajtos I am sorry, I cherry picked the commit and brought over the test.
You are right this is not related, and I will just omit the test for 3.X  branch